src/Twitter.php

```diff
@@ -803,14 +804,21 @@ public function friendshipsCreate($id, array $params = []) : Response
      *
      * Returns the next cursor if there are more to be returned.
      *
-     * @param int|string $id
+     * $id may be one of any of the following:
+     *
+     * - a single user identifier
+     * - a single screen name
+     * - a list of user identifiers
```

Comment:
Breaking 1:1 naming is not a problem, but introducing mixed endpoints is.

The consumer doesn't break anyway: if they need to use multi-value APIs, they just use the specialised method.

This is a limitation of PHP itself, since method overloading isn't allowed.

Regardless of BC, adding an union type is a problem.